### DISCUSSION TITLE
Cloning an Image to an extended part of the canvas

### DISCUSSION THREAD

**Discussion by awirthy** [ImageMagick/ImageMagick#7109] [Digital Image Processing] [answered]
How do I clone an image (![AA  103](https://github.com/ImageMagick/ImageMagick/assets/968090/57222aef-b692-4702-a462-4b56139d1ae7)) to an extended part of the same image so that it looks like this (![AA  103 2up](https://github.com/ImageMagick/ImageMagick/assets/968090/4d154ad9-2efb-4a5c-bb05-8e41aebea669))

Thanks in advance.
Aaron

**Comment by snibgo** [comment]
How about:
```
magick in.png ( +clone ) +append +repage out.png
```

**Comment by fmw42** [comment, marked as answer]
Perhaps you want

`magick image.jpg +duplicate +append +repage result.jpg`

Please always specify your IM version and platform as syntax may vary

**Reply by awirthy** [reply]
Thanks. This worked perfectly.
